apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -104,7 +103,7 @@ const RootStackNavigation = ({ initialRouteName }: RootStackNavigationProps) =>
                 initialRouteName={initialRouteName || 'LandingScreen'}
                 screenOptions={{ headerShown: false }}
               >
-                <RootStack.Group screenOptions={{ cardStyleInterpolator: CardStyleInterpolators.forFadeFromCenter }}>
+                <RootStack.Group screenOptions={{ animation: 'fade' }}>
```

Comment:
Maybe switching back to fade from center would fix most of the glitches?

Reply:
I played a bit with different values for this (and also moving this prop to like 104 and it's only the `default` animation and the `simple_push` that create the glitch.

Reply:
Fixed in the follow up PR!